src/man/sssd.conf.5.xml

```diff
@@ -161,7 +161,9 @@
                         <para>
                             Timeout in seconds between heartbeats for this
                             service. This is used to ensure that the process
-                            is alive and capable of answering requests.
+                            is alive and capable of answering requests. Note
+                            after three missed heartbeats process will self
```

Comment:
I'm not a native English speaker, but I do believe that we're missing some connectors in the sentence (please, check the bold words below.

"Note **that** after three missed heartbeats **the** process will ... " instead of "Note after three missed process will self".